apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -85,20 +103,27 @@ export default class VirtualizedDetailView extends Component {
     );
   };
 
-  getColumnWidth({index}) {
-    return columnWidths[index];
-  }
+  getColumnWidth = ({index}) => {
+    const {scriptData} = this.props;
+    if (index === 0) {
+      return 150;
```

Comment:
can we make 150 and 50 `NAME_COLUMN_WIDTH` and `BUBBLE_WIDTH` constants at the top of the file?